lib/feature/browser/view/browser_view.dart

```diff
@@ -1,7 +1,4 @@
-import 'package:app/app/router/router.dart';
 import 'package:flutter/material.dart';
-import 'package:go_router/go_router.dart';
-import 'package:ui_components_lib/ui_components_lib.dart';
 
 class BrowserView extends StatefulWidget {
```

Comment:
Maybe remove this widget because it doesn't contains anything

Reply:
`BrowserView` is a 'View' widget for `BrowserPage` that uses as builder for `StatefulShellRoute`. Theoretically we can remove `BrowserView` and render ShellRoute's children directly in `BrowserPage`, but we already have consideration that tell us to use separated `*View` widget to render widgets. So, this is just for uniformity of implementations.